### PR TITLE
Fix closing included views

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #2048 & #2058

### CODE REVIEW

**Comment by siemiatj** [comment]
This is ready to roview. I'm working on tests but will create a separate PR.

File: cypress/integration/manufacturing_order_spec.js [hunk 1]
```diff
@@ -12,4 +12,45 @@ describe('Manufacturing order Test', function() {
 
     cy.get('.form-field-Barcode').should('exist');
   });
+
+  it('Test if HU editor opens and closes', function() {
+    cy.visit('/window/53009');
+
+    cy.get('.table-flex-wrapper-row')
+      .find('tbody tr').eq(0)
+      .should('exist');
```
Comment (teosarca) [inline cypress/integration/manufacturing_order_spec.js]: @siemiatj @metas-ts  IMHO we shall get rid of things like
```
.get('.table-flex-wrapper-row')
.find('tbody tr')
```

Those names are implementation details which a regular test writer shall not be aware of.
More having things like that makes our tests uber-fragile.
wdyt?

Comment (metas-ts) [reply cypress/integration/manufacturing_order_spec.js]: totally agree

Comment (siemiatj) [reply cypress/integration/manufacturing_order_spec.js]: It's all true but I don't have the capacity to write custom commands for every operation atm :) It's way faster for me to check console, see what element I need and quickly use this in test.

**Pull request merged**